android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
+#Fri Sep 08 21:01:35 IST 2023
+networkTimeout=10000
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-all.zip
-networkTimeout=10000
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```

Comment:
Why these changes?

Reply:
Actually, I built it locally, so these changes were made automatically by the command